examples/README.md

```diff
@@ -1,2 +1,5 @@
 # Examples
-Our examples are organized by __target__.
\ No newline at end of file
+Our examples are organized by __target__ language:
+
+* [C examples](C/README.md)
+* [C++ examples](Cpp/README.md)
```

Comment:
I prefer the modularity given by the extra level of indirection.
Also, if we flatten it, it will make the imbalance between the number of examples for different targets very obvious.